tests/baselines/reference/enumBasics.js

```diff
@@ -145,7 +145,7 @@ var E8;
 var E9;
 (function (E9) {
     E9[E9["A"] = 0] = "A";
-    E9[E9["B"] = 0] = "B";
+    E9["B"] = E9.A;

```

Comment:
```
E9[E9.A] // "A"
E9[E9.B] // "A"
```
That's what I would expect, this pattern works well with string enums #3192 

Don't think the current behavior is more correct where the last declared member overrides the mapping:
```
E9[E9.A] // "B"
E9[E9.B] // "B"
```